convert_zdf.py

```diff
@@ -0,0 +1,150 @@
+"""Convert ZDF point cloud to the available formats.
+
+Example: $ python -u convert_zdf.py --ply Zivid3D.zdf
+
+Available formats:
+    .ply - Polygon File Format
+    .csv,.txt - [X, Y, Z, r, g, b, Contrast]
+    .png,.jpg,.bmp,.tiff - 2D RGB image
+"""
+
+import argparse
+from pathlib import Path
+import numpy as np
+import cv2
+import zivid
+
+
+def _options():
+    parser = argparse.ArgumentParser(
+        description="Convert from ZDF to your preferred format\
+            \nExample:\n\t $ python -u convert_zdf.py --ply Zivid3D.zdf",
+        formatter_class=argparse.RawTextHelpFormatter,
+    )
+
+    parser.add_argument("filename", help="File to convert", default="Zivid3D.zdf")
+    group = parser.add_mutually_exclusive_group(required=True)
+    group.add_argument("--ply", action="store_true", help="Convert to PLY")
+    group.add_argument("--csv", action="store_true", help="Convert to CSV")
+    group.add_argument("--txt", action="store_true", help="Convert to text")
+    group.add_argument(
+        "--png",
+        action="store_true",
+        help="Convert to Portable Network Graphics (3D->2D)",
+    )
+    group.add_argument("--jpg", action="store_true", help="Convert to JPEG (3D->2D)")
```

Comment:
can you move this to below --txt and --tiff below this one? Just some minor formatting fixes that black does not detect